File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/Buttons_View.cpp

#include "Buttons_View.h"
#include "Buttons_View.h"
#include <string>

namespace widgetassertions
{
  std::string Buttons_View::getOkButtonText( ) 
  {
    return this->OkButtonText;
  }
  bool Buttons_View::getIsOkButtonEnabled( ) 
  {
    return this->isOkButtonEnabled;
  }
  bool Buttons_View::getIsOkButtonVisibility( ) 
  {
    return this->isOkButtonVisibility;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/Buttons_View.cs
/// <filename>
///     Buttons_View.cs
/// </filename>


namespace widgetassertions
{
    public class Buttons_View
  {
        private string OkButtonText;
     
        public     virtual string getOkButtonText()
    {
      return this.OkButtonText;
    }
     
        private bool isOkButtonEnabled;
     
        public     virtual bool getIsOkButtonEnabled()
    {
      return this.isOkButtonEnabled;
    }
     
        private bool isOkButtonVisible;
     
        public     virtual bool getIsOkButtonVisible()
    {
      return this.isOkButtonVisible;
    }
  }
}

